src/search/merge_and_shrink/merge_and_shrink_algorithm.cc

```diff
@@ -83,6 +83,9 @@ void MergeAndShrinkAlgorithm::dump_options() const {
             << shrink_threshold_before_merge << endl;
         log << endl;
 
+        shrink_strategy->dump_options(log);
+        log << endl;
```

Comment:
Is this here intentionally? Is it related to the issue? I don't mind adding it if you want to add it. Should it be mentioned in the change log then? More generally, I think you don't have a change log entry yet?

Reply:
(BTW, I'm not convinced that adding a blank line is great. With structured logging formats this looks weird.)

Reply:
This was removed accidentally in another issue. I don't think that there should be a change log entry. The empty lines are an attempt to structuring, but yes, with the prefix printed, it is probably nicer (and a lot easier) to not use any empty lines for structuring.

Reply:
But I wouldn't touch this now -- this is done at several places, and I don't dislike it everywhere . For example, having an empty line between the output of each iteration of M&S doesn't seem that bad to me, but we can certainly discuss this.

Reply:
I think with this kind of logging format, it's weird with or without the prefix line. But feel free to leave this as is if it's not new to this issue.